### PR TITLE
UI/15399783 side panel nav tabs

### PR DESCRIPTION
**Pull request by KatarzynaStefanowiczPagepro**
## Description

- [] TODO write description

**New stuff** ✨

* New stuff goes here

**Changes** 🏗

* Changes go here

**Deletions** ⚰️

* Deletions go here

## TODO

- [ ] Add TODOs

(Resolves | Contributes to) #31415

### CODE REVIEW

**Comment by KatarzynaStefanowiczPagepro** [comment]
https://tw.auroracreation.com/app/tasks/15399783
Added routing and prepared page layout shared across all tabs
image

**Review by CzarekDryl — APPROVED** [review]

File: src/components/frame/Extensions/layouts/ExtensionLayout.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        loadingText={{ id: 'extensionsPage.loading' }}
+        appearance={{ theme: 'primary', size: 'massive' }}
+      />
+    );
+  }
+
+  return (
+    <div>
+      <Header />
+      {/* @TODO: Remove wallet component when we have a proper wallet */}
```
Comment (krawczykkatarzyna) [inline src/components/frame/Extensions/layouts/ExtensionLayout.tsx:30]: we have wallet component, can you add it ?

File: src/components/frame/Extensions/pages/AdvancedPage/AdvancedPage.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React, { FC } from 'react';
+import { useExtensionsData } from '~hooks';
+import { SpinnerLoader } from '~shared/Preloaders';
+
+const displayName = 'frame.Extensions.pages.AdvancedPage';
+
+const AdvancedPage: FC = () => {
+  const { loading } = useExtensionsData();
```
Comment (krawczykkatarzyna) [inline src/components/frame/Extensions/pages/AdvancedPage/AdvancedPage.tsx:8]: please create separate component for displaying spinner and import it in all places

File: src/components/frame/Extensions/pages/ExtensionPage/ExtensionPage.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+const ExtensionPage: FC = () => {
+  const { loading, availableExtensionsData, installedExtensionsData } = useExtensionsData();
+  const { formatMessage } = useIntl();
+
+  const allExtensions = useMemo(
+    () => [...availableExtensionsData, ...installedExtensionsData],
+    [availableExtensionsData, installedExtensionsData],
+  );
+
+  if (loading) {
```
Comment (krawczykkatarzyna) [inline src/components/frame/Extensions/pages/ExtensionPage/ExtensionPage.tsx:18]: the same, import created component

File: src/components/frame/Extensions/pages/IncorporationPage/IncorporationPage.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import React, { FC } from 'react';
+import { useExtensionsData } from '~hooks';
+import { SpinnerLoader } from '~shared/Preloaders';
+
+const displayName = 'frame.Extensions.pages.IncorporationPage';
+
+const IncorporationPage: FC = () => {
+  const { loading } = useExtensionsData();
+
+  if (loading) {
```
Comment (krawczykkatarzyna) [inline src/components/frame/Extensions/pages/IncorporationPage/IncorporationPage.tsx:10]: the same, import created component

File: src/components/frame/Extensions/pages/IntegrationsPage/IntegrationsPage.tsx [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+import { SpinnerLoader } from '~shared/Preloaders';
+
+const displayName = 'frame.Extensions.pages.IntegrationsPage';
+
+const IntegrationsPage: FC = () => {
+  const { loading } = useExtensionsData();
+
+  if (loading) {
+    return (
+      <SpinnerLoader
```
Comment (krawczykkatarzyna) [inline src/components/frame/Extensions/pages/IntegrationsPage/IntegrationsPage.tsx:12]: the same

File: src/components/frame/Extensions/pages/ReputationPage/ReputationPage.tsx [hunk 6]
```diff
@@ -0,0 +1,24 @@
+import React, { FC } from 'react';
+import { useExtensionsData } from '~hooks';
+import { SpinnerLoader } from '~shared/Preloaders';
+
+const displayName = 'frame.Extensions.pages.ReputationPage';
+
+const ReputationPage: FC = () => {
+  const { loading } = useExtensionsData();
+
```
Comment (krawczykkatarzyna) [inline src/components/frame/Extensions/pages/ReputationPage/ReputationPage.tsx:9]: the same

**Pull request merged**